### ISSUE TITLE
[Modcompat]: Unable to sleep during the day

### ISSUE THREAD

**Issue by ghost** [TeamLapen/Vampirism#1170]
### Minecraft Version

1.19.x

### Forge Version

43.2.3

### Vampirism Version

Vampirism-1.19.2-1.9.0

### Incompatible mod

Unknown content, but certainly not Vampirism.

### Singleplayer/Multiplayer

Singleplayer

### What happened?

Unable to sleep during the day

### Reproduce steps

https://imgloc.com/i/CjIvb

### Relevant log output

```shell
https://t.wss.ink/f/a8lm9dwg4qs
```

**Comment by Cheaterpaul** [comment]
From what i can, there is another mod present that does not allow you to sleep or mishandles the time check.

Do you know of any mod you use that adds some sleeping functionality as well? E.g other sleeping blocks

**Comment by ghost** [comment]
[Radon (by Asek3)](https://www.curseforge.com/minecraft/mc-mods/radon)
[Classic Bars (by tfarecnim)](https://www.curseforge.com/minecraft/mc-mods/classic-bars)
[Configured (by MrCrayfish)](https://www.curseforge.com/minecraft/mc-mods/configured)
[Doggy Talents (by percivalalb)](https://www.curseforge.com/minecraft/mc-mods/doggy-talents)
[Raven Coffee (by TheWandererRaven)](https://www.curseforge.com/minecraft/mc-mods/raven-coffee)
[Entity Collision FPS Fix (by Corgi_Taco)](https://www.curseforge.com/minecraft/mc-mods/entity-collision-fps-fix)
[Iceberg [Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/iceberg)
[MrCrayfish's Furniture Mod (by MrCrayfish)](https://www.curseforge.com/minecraft/mc-mods/mrcrayfish-furniture-mod)
[Multi-Piston (by H3lay)](https://www.curseforge.com/minecraft/mc-mods/multi-piston)
[Neapolitan (by TeamAbnormals)](https://www.curseforge.com/minecraft/mc-mods/neapolitan)
[ChoiceTheorem's Overhauled Village (by ChoiceTheorem)](https://www.curseforge.com/minecraft/mc-mods/choicetheorems-overhauled-village)
[Buzzier Bees (by TeamAbnormals)](https://www.curseforge.com/minecraft/mc-mods/buzzier-bees)
[Nether's Delight (by lumpazl)](https://www.curseforge.com/minecraft/mc-mods/nethers-delight)
[Random Sheep Colours (by Serilum)](https://www.curseforge.com/minecraft/mc-mods/random-sheep-colours)
[Nyf's Spiders (by theNyfaria)](https://www.curseforge.com/minecraft/mc-mods/nyfs-spiders)
[Xaero's Minimap (by xaero96)](https://www.curseforge.com/minecraft/mc-mods/xaeros-minimap)
[Oculus (by Asek3)](https://www.curseforge.com/minecraft/mc-mods/oculus)
[Domum Ornamentum (by OrionOnline)](https://www.curseforge.com/minecraft/mc-mods/domum-ornamentum)
[Moonlight Lib (by MehVahdJukaar)](https://www.curseforge.com/minecraft/mc-mods/selene)
[Exotic Birds (by ThePavoReality)](https://www.curseforge.com/minecraft/mc-mods/exotic-birds)
[Comforts (Fabric/Forge/Quilt) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/comforts)
[Xaero's World Map (by xaero96)](https://www.curseforge.com/minecraft/mc-mods/xaeros-world-map)
[Carry On (by Tschipp)](https://www.curseforge.com/minecraft/mc-mods/carry-on)
[Game Menu Remove GFARB (by morimori0317)](https://www.curseforge.com/minecraft/mc-mods/game-menu-remove-gfarb)
[Friendly Fire (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/friendly-fire)
[Inventory HUD+ (by dmitrylovin)](https://www.curseforge.com/minecraft/mc-mods/inventory-hud-forge)
[MCA Reborn [Fabric/Forge] (by jahx_senpoopie)](https://www.curseforge.com/minecraft/mc-mods/minecraft-comes-alive-reborn)
[Akashic Tome (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/akashic-tome)
[Skewers Updated (by ACGaming56)](https://www.curseforge.com/minecraft/mc-mods/skewers-updated)
[Domestication Innovation (by sbom_xela)](https://www.curseforge.com/minecraft/mc-mods/domestication-innovation)
[Drink Beer Refill (by dragons_plus)](https://www.curseforge.com/minecraft/mc-mods/drink-beer-refill)
[Abnormals Delight (by TeamAbnormals)](https://www.curseforge.com/minecraft/mc-mods/abnormals-delight)
[Advancement Plaques [Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/advancement-plaques)
[Towns and Towers (by Biban_Auriu)](https://www.curseforge.com/minecraft/mc-mods/towns-and-towers)
[Not Enough Animations (by tr9zw)](https://www.curseforge.com/minecraft/mc-mods/not-enough-animations)
[Nutritious Milk (by Serilum)](https://www.curseforge.com/minecraft/mc-mods/nutritious-milk)
[Legendary Tooltips [Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/legendary-tooltips)
[Placebo (by Shadows_of_Fire)](https://www.curseforge.com/minecraft/mc-mods/placebo)
[Wavey Capes (by tr9zw)](https://www.curseforge.com/minecraft/mc-mods/waveycapes)
[Mooshroom Spawn (by Serilum)](https://www.curseforge.com/minecraft/mc-mods/mooshroom-spawn)
[Traveler's Backpack (by Tiviacz1337)](https://www.curseforge.com/minecraft/mc-mods/travelers-backpack)
[Falling Leaves (Forge) (by Cheaterpaul)](https://www.curseforge.com/minecraft/mc-mods/falling-leaves-forge)
[Gobber Delight (A Farmer's Delight Add-on) (by nocubeyt)](https://www.curseforge.com/minecraft/mc-mods/gobber-delight-a-farmers-delight-add-on)
[Item Highlighter [Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/item-highlighter)
[Snow! Real Magic! ⛄ (Forge) (by Snownee_)](https://www.curseforge.com/minecraft/mc-mods/snow-real-magic)
[The Twilight Forest (by Benimatic)](https://www.curseforge.com/minecraft/mc-mods/the-twilight-forest)
[Game Menu Mod Option [Forge] (by morimori0317)](https://www.curseforge.com/minecraft/mc-mods/gamemenumodoption)
[Dragon Mounts: Legacy (by UnitKay9)](https://www.curseforge.com/minecraft/mc-mods/dragon-mounts-legacy)
[Roughly Enough Items Fabric/Forge (REI) (by shedaniel)](https://www.curseforge.com/minecraft/mc-mods/roughly-enough-items)
[BlockUI (by H3lay)](https://www.curseforge.com/minecraft/mc-mods/blockui)
[Curios Quark Oddities Backpack (by 6LeoY)](https://www.curseforge.com/minecraft/mc-mods/curios-quark-oddities-backpack)
[Frozen Delight (A Farmer's Delight Add-on) (by nocubeyt)](https://www.curseforge.com/minecraft/mc-mods/frozen-delight)
[Saturn (by AbdElAziz333)](https://www.curseforge.com/minecraft/mc-mods/saturn)
[Creeper Overhaul (by joosh_7889)](https://www.curseforge.com/minecraft/mc-mods/creeper-overhaul)
[SwingThroughGrass (by exidex)](https://www.curseforge.com/minecraft/mc-mods/swingthroughgrass)
[Delightful (by brdbrn)](https://www.curseforge.com/minecraft/mc-mods/delightful)
[Paragliders (by Tictim)](https://www.curseforge.com/minecraft/mc-mods/paragliders)
[Quests Additions (Forge) (by NaturaSpell)](https://www.curseforge.com/minecraft/mc-mods/quests-additions)
[Puzzles Lib (by Fuzs_)](https://www.curseforge.com/minecraft/mc-mods/puzzles-lib)
[YUNG's Better Desert Temples (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-desert-temples)
[Trample Stopper (by UnRealDinnerbone)](https://www.curseforge.com/minecraft/mc-mods/trample-stopper)
[Modern UI (by BloCamLimb)](https://www.curseforge.com/minecraft/mc-mods/modern-ui)
[TerraBlender (Forge) (by TheAdubbz)](https://www.curseforge.com/minecraft/mc-mods/terrablender)
[Greek Fantasy (by skyjay1)](https://www.curseforge.com/minecraft/mc-mods/greek-fantasy)
[Yeetus Experimentus (by Sunekaer)](https://www.curseforge.com/minecraft/mc-mods/yeetusexperimentus)
[Miner's Helmet (by ModdingLegacy)](https://www.curseforge.com/minecraft/mc-mods/miners-helmet)
[AutoRegLib (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/autoreglib)
[AI Improvements (by QueenOfMissiles)](https://www.curseforge.com/minecraft/mc-mods/ai-improvements)
[Caelus API (Forge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/caelus)
[Canes Wonderful Spiders (by caneswonderfulaccount)](https://www.curseforge.com/minecraft/mc-mods/canes-wonderful-spiders)
[Cooking for Blockheads (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/cooking-for-blockheads)
[Collective (by Serilum)](https://www.curseforge.com/minecraft/mc-mods/collective)
[Pluto (by AbdElAziz333)](https://www.curseforge.com/minecraft/mc-mods/pluto)
[TofuCraftReload (by sz0999312)](https://www.curseforge.com/minecraft/mc-mods/tofucraftreload)
[Entity Culling Fabric/Forge (by tr9zw)](https://www.curseforge.com/minecraft/mc-mods/entityculling)
[Integrated Dungeons and Structures (by CraisinLord)](https://www.curseforge.com/minecraft/mc-mods/idas)
[HT's TreeChop (by hammertater)](https://www.curseforge.com/minecraft/mc-mods/treechop)
[REI Plugin Compatibilities (REIPC) (by shedaniel)](https://www.curseforge.com/minecraft/mc-mods/roughly-enough-items-hacks)
[LibX (by noeppinoeppi)](https://www.curseforge.com/minecraft/mc-mods/libx)
[Aquaculture 2 (by Shadowclaimer)](https://www.curseforge.com/minecraft/mc-mods/aquaculture)
[Savage & Ravage (by TeamAbnormals)](https://www.curseforge.com/minecraft/mc-mods/savage-and-ravage)
[Vampirism - Become a vampire! (by maxanier)](https://www.curseforge.com/minecraft/mc-mods/vampirism-become-a-vampire)
[Better Villages - Forge (by jtl_elisa)](https://www.curseforge.com/minecraft/mc-mods/better-village-forge)
[Structurize (by H3lay)](https://www.curseforge.com/minecraft/mc-mods/structurize)
[Polymorph (Forge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/polymorph)
[Paladin's Furniture (by UnlikePaladin)](https://www.curseforge.com/minecraft/mc-mods/paladins-furniture)
[More Babies (Forge & Fabric) (by mammut53m)](https://www.curseforge.com/minecraft/mc-mods/more-babies)
[Culinary Construct (Forge/Fabric) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/culinary-construct)
[Let's do Wine! (by satisfyL)](https://www.curseforge.com/minecraft/mc-mods/lets-do-wine)
[Better With Minecolonies [Forge] (by Kreezxil)](https://www.curseforge.com/minecraft/mc-mods/better-with-minecolonies)
[Alex's Mobs (by sbom_xela)](https://www.curseforge.com/minecraft/mc-mods/alexs-mobs)
[project-576825 (by player_in_distress)](https://www.curseforge.com/minecraft/mc-mods/project-576825)
[TexTrue's Rubidium Options (by TexTrue)](https://www.curseforge.com/minecraft/mc-mods/textrues-rubidium-options)
[Deeper and Darker (by KyaniteMods)](https://www.curseforge.com/minecraft/mc-mods/deeperdarker)
[Bookshelf (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/bookshelf)
[Untitled Duck Mod (Forge) (by Okabintaro)](https://www.curseforge.com/minecraft/mc-mods/untitled-duck-mod-forge)
[Camp Chair (by dmitrylovin)](https://www.curseforge.com/minecraft/mc-mods/camp-chair)
[Blueprint (by TeamAbnormals)](https://www.curseforge.com/minecraft/mc-mods/blueprint)
[Food Enhancements (by ISenseHostility)](https://www.curseforge.com/minecraft/mc-mods/food-enhancements)
[Smooth Boot (Reloaded) (by AbdElAziz333)](https://www.curseforge.com/minecraft/mc-mods/smooth-boot-reloaded)
[Roost Ultimate (by TarantelOfficial)](https://www.curseforge.com/minecraft/mc-mods/roost-ultimate)
[PizzaCraft (by Tiviacz1337)](https://www.curseforge.com/minecraft/mc-mods/pizzacraft)
[When Dungeons Arise - Forge! (by Aureljz)](https://www.curseforge.com/minecraft/mc-mods/when-dungeons-arise)
[Mouse Tweaks (by YaLTeR)](https://www.curseforge.com/minecraft/mc-mods/mouse-tweaks)
[Sophisticated Core (by P3pp3rF1y)](https://www.curseforge.com/minecraft/mc-mods/sophisticated-core)
[Citadel (by sbom_xela)](https://www.curseforge.com/minecraft/mc-mods/citadel)
[Origins: Classes (Forge) (by LimonBlaze)](https://www.curseforge.com/minecraft/mc-mods/origins-classes-forge)
[Library Ferret - Forge (by jtl_elisa)](https://www.curseforge.com/minecraft/mc-mods/library-ferret-forge)
[Prism [Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/prism-lib)
[AttributeFix (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/attributefix)
[Ravage & Cabbage (by superlord_9362)](https://www.curseforge.com/minecraft/mc-mods/ravage-and-cabbage)
[Supplementaries (by MehVahdJukaar)](https://www.curseforge.com/minecraft/mc-mods/supplementaries)
[Architectury API (Fabric/Forge) (by shedaniel)](https://www.curseforge.com/minecraft/mc-mods/architectury-api)
[Nether's Exoticism (by Furti_Two)](https://www.curseforge.com/minecraft/mc-mods/nethers-exoticism)
[FastFood Delight (by akaneo2)](https://www.curseforge.com/minecraft/mc-mods/fastfood-delight)
[Beekeeper (by MacTso)](https://www.curseforge.com/minecraft/mc-mods/beekeeper)
[FerriteCore (Forge) (by malte0811)](https://www.curseforge.com/minecraft/mc-mods/ferritecore)
[Fruit Trees 🍊 (by Snownee_)](https://www.curseforge.com/minecraft/mc-mods/fruit-trees)
[Seasonals (by catscythee)](https://www.curseforge.com/minecraft/mc-mods/seasonals)
[Diagonal Fences (by Fuzs_)](https://www.curseforge.com/minecraft/mc-mods/diagonal-fences)
[Quark Oddities (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/quark-oddities)
[Immersive Weathering [FORGE] (by OrdanaryMods)](https://www.curseforge.com/minecraft/mc-mods/immersive-weathering-forge)
[Patchouli (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/patchouli)
[Item Filters (by LatvianModder)](https://www.curseforge.com/minecraft/mc-mods/item-filters)
[Alex's Delight (by NCP_Bails)](https://www.curseforge.com/minecraft/mc-mods/alexs-delight)
[Sit (by bl4ckscor3)](https://www.curseforge.com/minecraft/mc-mods/sit)
[Better Title Screen (by Girafi)](https://www.curseforge.com/minecraft/mc-mods/better-title-screen)
[Curios API (Forge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/curios)
[Ecologics (by SameDifferent)](https://www.curseforge.com/minecraft/mc-mods/ecologics)
[Hostile Mobs and Girls (HMaG) (by Mechalopa)](https://www.curseforge.com/minecraft/mc-mods/hostile-mobs-and-girls)
[Guide-API Village and Pillage (by maxanier)](https://www.curseforge.com/minecraft/mc-mods/guide-api-village-and-pillage)
[Balm (Forge Edition) (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/balm)
[FTB Teams (Forge) (by FTB)](https://www.curseforge.com/minecraft/mc-mods/ftb-teams-forge)
[Roughly Enough Professions (REP) (by Mrbysco)](https://www.curseforge.com/minecraft/mc-mods/roughly-enough-professions-rep)
[Naturalist (by Starfish_Studios)](https://www.curseforge.com/minecraft/mc-mods/naturalist)
[Just Enough Effect Descriptions (JEED) (by MehVahdJukaar)](https://www.curseforge.com/minecraft/mc-mods/just-enough-effect-descriptions-jeed)
[Stalwart Dungeons (by Furti_Two)](https://www.curseforge.com/minecraft/mc-mods/stalwart-dungeons)
[YUNG's Bridges (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-bridges)
[Rotten Creatures (by fusionstudiomc)](https://www.curseforge.com/minecraft/mc-mods/rotten-creatures)
[YUNG's API (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-api)
[Goblin Traders (by MrCrayfish)](https://www.curseforge.com/minecraft/mc-mods/goblin-traders)
[Cosmetic Armor Reworked (by LainMI)](https://www.curseforge.com/minecraft/mc-mods/cosmetic-armor-reworked)
[FTB Library (Forge) (by FTB)](https://www.curseforge.com/minecraft/mc-mods/ftb-library-forge)
[Werewolves - Become a Beast! (by Cheaterpaul)](https://www.curseforge.com/minecraft/mc-mods/werewolves-become-a-beast)
[Nature's Compass (by Chaosyr)](https://www.curseforge.com/minecraft/mc-mods/natures-compass)
[Better Advancements (by way2muchnoise)](https://www.curseforge.com/minecraft/mc-mods/better-advancements)
[Just Enough Resources (JER) (by way2muchnoise)](https://www.curseforge.com/minecraft/mc-mods/just-enough-resources-jer)
[Create Cafe (by 1mphuls3)](https://www.curseforge.com/minecraft/mc-mods/create-cafe)
[FTB Quests (Forge) (by FTB)](https://www.curseforge.com/minecraft/mc-mods/ftb-quests-forge)
[Fast Leaf Decay (by olafskiii)](https://www.curseforge.com/minecraft/mc-mods/fast-leaf-decay)
[Earth Mobs Mod (by Derec__)](https://www.curseforge.com/minecraft/mc-mods/minecraft-earth-mod)
[Visual Workbench (by Fuzs_)](https://www.curseforge.com/minecraft/mc-mods/visual-workbench)
[Waystones (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/waystones)
[Random Shulker Colours (by Serilum)](https://www.curseforge.com/minecraft/mc-mods/random-shulker-colours)
[Storage Drawers (by Texelsaur)](https://www.curseforge.com/minecraft/mc-mods/storage-drawers)
[Oh The Biomes You'll Go (by AOCAWOL)](https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go)
[Smaller Nether Portals (by Serilum)](https://www.curseforge.com/minecraft/mc-mods/smaller-nether-portals)
[YUNG's Better Strongholds (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-strongholds)
[Create Confectionery (by Furti_Two)](https://www.curseforge.com/minecraft/mc-mods/create-confectionery)
[Explorer's Compass (by Chaosyr)](https://www.curseforge.com/minecraft/mc-mods/explorers-compass)
[Vein Mining (Fabric/Forge/Quilt) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/vein-mining)
[Origins (Forge) (by EdwinMindcraft)](https://www.curseforge.com/minecraft/mc-mods/origins-forge)
[ShulkerBoxTooltip [Fabric/Forge] (by MisterPeModder)](https://www.curseforge.com/minecraft/mc-mods/shulkerboxtooltip)
[YUNG's Better Dungeons (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-dungeons)
[Just Enough Characters (by towdium)](https://www.curseforge.com/minecraft/mc-mods/just-enough-characters)
[Quark (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/quark)
[Vampirism Integrations (by maxanier)](https://www.curseforge.com/minecraft/mc-mods/vampirism-integrations)
[Boatload (by TeamAbnormals)](https://www.curseforge.com/minecraft/mc-mods/boatload)
[Jade 🔍 (by Snownee_)](https://www.curseforge.com/minecraft/mc-mods/jade)
[Keyboard Wizard (by mrnerdy42)](https://www.curseforge.com/minecraft/mc-mods/keyboard-wizard)
[Snow Under Trees (by bl4ckscor3)](https://www.curseforge.com/minecraft/mc-mods/snow-under-trees)
[Passable Foliage 🌳 (Forge) (by Snownee_)](https://www.curseforge.com/minecraft/mc-mods/passable-foliage)
[Create (by simibubi)](https://www.curseforge.com/minecraft/mc-mods/create)
[Croptopia (by thethonk)](https://www.curseforge.com/minecraft/mc-mods/croptopia)
[Caupona (by khjxiaogu)](https://www.curseforge.com/minecraft/mc-mods/caupona)
[Canary (by AbdElAziz333)](https://www.curseforge.com/minecraft/mc-mods/canary)
[Sky Villages [Forge] (by y4z0n)](https://www.curseforge.com/minecraft/mc-mods/sky-villages-forge)
[Let Me Despawn (by frikinjay1)](https://www.curseforge.com/minecraft/mc-mods/let-me-despawn)
[Gobber [FORGE] (by kwpugh)](https://www.curseforge.com/minecraft/mc-mods/gobber)
[Rubidium (by Asek3)](https://www.curseforge.com/minecraft/mc-mods/rubidium)
[YUNG's Extras (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-extras)
[Sophisticated Backpacks (by P3pp3rF1y)](https://www.curseforge.com/minecraft/mc-mods/sophisticated-backpacks)
[Effective (Forge) (by BottomTextDanny)](https://www.curseforge.com/minecraft/mc-mods/effective-forge)
[Just Enough Advancements (JEA) (by MelanX)](https://www.curseforge.com/minecraft/mc-mods/jea)
[Frozen Up (by ninnih_)](https://www.curseforge.com/minecraft/mc-mods/frozen-up)
[Jade Addons (Forge) (by Snownee_)](https://www.curseforge.com/minecraft/mc-mods/jade-addons)
[Controlling (by Jaredlll08)](https://www.curseforge.com/minecraft/mc-mods/controlling)
[Serene Seasons (by TheAdubbz)](https://www.curseforge.com/minecraft/mc-mods/serene-seasons)
[Spice of Life: Carrot Edition (by lordcazsius)](https://www.curseforge.com/minecraft/mc-mods/spice-of-life-carrot-edition)
[MineColonies (by H3lay)](https://www.curseforge.com/minecraft/mc-mods/minecolonies)
[Farmer's Delight (by vectorwing)](https://www.curseforge.com/minecraft/mc-mods/farmers-delight)
[Raven Brews Core (by TheWandererRaven)](https://www.curseforge.com/minecraft/mc-mods/raven-brews-core)
[Corpse (by henkelmax)](https://www.curseforge.com/minecraft/mc-mods/corpse)
[Waystone Waypoint Maker (by hextun01)](https://www.curseforge.com/minecraft/mc-mods/waymaker)
[Earth Mobs (by bagu_chan500)](https://www.curseforge.com/minecraft/mc-mods/earth-mobs)
[Productive Bees (by LobsterJonn)](https://www.curseforge.com/minecraft/mc-mods/productivebees)
[Simple Jams (by timeninja1)](https://www.curseforge.com/minecraft/mc-mods/simple-jams)
[Dungeon Crawl (by xiroc_)](https://www.curseforge.com/minecraft/mc-mods/dungeon-crawl)
[Better Animations Collection (by Fuzs_)](https://www.curseforge.com/minecraft/mc-mods/better-animations-collection)
[Castle in the Sky (by songxia23)](https://www.curseforge.com/minecraft/mc-mods/castle-in-the-sky-the-fairytale-of-laputa)
[YUNG's Better Ocean Monuments (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-ocean-monuments)
[GeckoLib (by ThanosGecko)](https://www.curseforge.com/minecraft/mc-mods/geckolib)
[Enchantment Descriptions (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/enchantment-descriptions)
[Biomes O' Plenty (by Forstride)](https://www.curseforge.com/minecraft/mc-mods/biomes-o-plenty)
[Enchanting Infuser (by Fuzs_)](https://www.curseforge.com/minecraft/mc-mods/enchanting-infuser-forge)
[Pineapple Delight (by someoneice)](https://www.curseforge.com/minecraft/mc-mods/pineapple-delight)
[Immersive FX (by y4z0n)](https://www.curseforge.com/minecraft/mc-mods/immersive-fx)
[Catalogue (by MrCrayfish)](https://www.curseforge.com/minecraft/mc-mods/catalogue)
[Just More Cakes! (by MincraftEinstein)](https://www.curseforge.com/minecraft/mc-mods/just-more-cakes)
[Additional Structures (by XxRexRaptorxX)](https://www.curseforge.com/minecraft/mc-mods/additional-structures)
[JER Integration (by AlasDiablo)](https://www.curseforge.com/minecraft/mc-mods/jer-integration)
[Kiwi 🥝 (Forge) (by Snownee_)](https://www.curseforge.com/minecraft/mc-mods/kiwi)
[YUNG's Better Mineshafts (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-mineshafts-forge)
[Cloth Config API (Fabric/Forge) (by shedaniel)](https://www.curseforge.com/minecraft/mc-mods/cloth-config)
[YUNG's Better Witch Huts (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-witch-huts)
[Better Rubidium Video Settings Button (by TexTrue)](https://www.curseforge.com/minecraft/mc-mods/brvsb)
[AppleSkin (by squeek502)](https://www.curseforge.com/minecraft/mc-mods/appleskin)
[Creeper Confetti (by SR2610)](https://www.curseforge.com/minecraft/mc-mods/creeper-confetti)
[Structory (by botanydev)](https://www.curseforge.com/minecraft/mc-mods/structory)
[Rubidium Extra (by dimadencep)](https://www.curseforge.com/minecraft/mc-mods/rubidium-extra)

**Comment by ghost** [comment]
I really didn't know that existed

**Comment by ghost** [comment]
I tested like crazy, but it seemed like no matter who I deleted, it kept changing,

**Comment by ghost** [comment]
Sorry for the trouble, but I really don't know which mod is the problem.

**Comment by Cheaterpaul** [comment]
From the list i can not really infer possible mods that could interfere with Vampirism.

You could try this to narrow down the problematic mod by doing the following:
(while doing so make sure to make a backup world or use a fresh one)

1. remove half of the mods
2. try sleeping again.
3. If you still can not sleep: go to step 1
4. If the problem is solved replace you mods with the ones you removed in step 1 and go to step 2

TLDR: divide you mods by 2 until the problem does no longer occur to rule mods out

**Comment by ghost** [comment]
> From the list i can not really infer possible mods that could interfere with Vampirism.
> 
> You could try this to narrow down the problematic mod by doing the following: (while doing so make sure to make a backup world or use a fresh one)
> 
> 1. remove half of the mods
> 2. try sleeping again.
> 3. If you still can not sleep: go to step 1
> 4. If the problem is solved replace you mods with the ones you removed in step 1 and go to step 2
> 
> TLDR: divide you mods by 2 until the problem does no longer occur to rule mods out

I've tried many times, but the results have been erratic.

**Comment by ghost** [comment]
> From the list i can not really infer possible mods that could interfere with Vampirism.
> 
> You could try this to narrow down the problematic mod by doing the following: (while doing so make sure to make a backup world or use a fresh one)
> 
> 1. remove half of the mods
> 2. try sleeping again.
> 3. If you still can not sleep: go to step 1
> 4. If the problem is solved replace you mods with the ones you removed in step 1 and go to step 2
> 
> TLDR: divide you mods by 2 until the problem does no longer occur to rule mods out

It takes a lot of time to become a vampire, Is there a quick way to become a vampire so we can experiment?

**Comment by Cheaterpaul** [comment]
you can use `/vampirism level vampirism:vampire 1` to become a vampire

**Comment by ghost** [comment]
Thank you. That's a lot faster.
I'll let you know when I get the result.

**Comment by ghost** [comment]
> 你可以用来成为吸血鬼`/vampirism level vampirism:vampire 1`

Mods can sleep when they are split, but not when they are all merged, which is weird.

**Comment by ghost** [comment]
> 你可以用来成为吸血鬼`/vampirism level vampirism:vampire 1`

SkyVillages-1.0.1-1.19-forge-release.jar
The target is relatively confirmed at present

**Comment by ghost** [comment]
> > 你可以用来成为吸血鬼`/vampirism level vampirism:vampire 1`
> 
> 天空村-1.0.1-1.19-锻造发布.jar 目标目前相对确定

Wrong result, still undetermined, same result when all mods load. Still can't sleep.

**Comment by Cheaterpaul** [comment]
This is most likely caused by Comforts https://github.com/illusivesoulworks/comforts/issues/122

**Comment by Cheaterpaul** [comment]
Should be fixed in new comforts versions
